### DISCUSSION TITLE
Hiding the text "Expand" from column

### DISCUSSION THREAD

**Discussion by davidloh85** [KevinVandy/material-react-table#797] [Q&A] [answered]
Hi, after setting enableExpandAll flag to false, there is a "Expand" label on the header.  Is there anyway to remove it?

![Screenshot 2023-11-07 at 11 26 51 AM](https://github.com/KevinVandy/material-react-table/assets/866373/365b9b2e-ef84-41de-8daa-365be78eacb8)

**Comment by nathan-fletcher-seagull-com** [comment, marked as answer]
displayColumnDefOptions={{
    "mrt-row-expand": {
        header: "",

has been working for me, though I don't know if that's the intended way to do it.

**Reply by davidloh85** [reply]
Thank you ! that works ! I guess i will go follow up and understand better how i can customise the columns
